### PR TITLE
💄 Refactor menu items

### PR DESCRIPTION
**Pull request by williamchong**
image

### CODE REVIEW

File: components/SiteMenu.vue [hunk 1]
```diff
@@ -53,74 +53,58 @@ function handleLinkClick () {
 
 const items = [
   {
-    label: 'Mint NFT',
+    label: 'NFT Book',
     links: [
       {
-        label: 'Mint NFT',
+        label: 'Mint New Books',
```
Comment (nwingt) [inline components/SiteMenu.vue:59]:
```suggestion
        label: 'Create New Books',
```

File: components/SiteMenu.vue [hunk 2]
```diff
@@ -53,74 +53,58 @@ function handleLinkClick () {
 
 const items = [
   {
-    label: 'Mint NFT',
+    label: 'NFT Book',
```
Comment (nwingt) [inline components/SiteMenu.vue:56]:
```suggestion
    label: 'Publish',
```

**Review by nwingt — COMMENTED** [review]
More layman friendly

File: components/SiteMenu.vue (hunk 2 above)
Comment (williamchong) [reply components/SiteMenu.vue:56]: I would argue in book press they would already know something about nft or wallet or blockchain, they need LIKE

File: components/SiteMenu.vue (hunk 1 above)
Comment (williamchong) [reply components/SiteMenu.vue:59]: same ui is used for restock though, not sure if any better words

Comment (williamchong) [reply components/SiteMenu.vue:59]: `Print` maybe

Comment (nwingt) [reply components/SiteMenu.vue:59]: I am following KDP
image

File: components/SiteMenu.vue (hunk 2 above)
Comment (nwingt) [reply components/SiteMenu.vue:56]: no strong opinion

**Review by nwingt — APPROVED** [review]

**Pull request merged**